src/ResourceManager/Common/Commands.ResourceManager.Common/Commands.ResourceManager.Common.csproj

```diff
@@ -221,6 +222,10 @@
       <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
       <Name>Commands.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Commands.Common.Authentication.ResourceManager\Commands.Common.Authentication.ResourceManager.csproj">
```

Comment:
We should not have a reference to Commands.ResourceManager.Common in this DLL, we may need to look at refactoring the location of this component, if this is required.

Reply:
Not sure why this got added, not using it in the code. Removed.